lexer.c

```diff
@@ -71,6 +76,12 @@ t_token *lexer(char **str)
 			}
 			else
 			{
+                // Check for invalid tokens :)
+                if (**str != '+' && **str != '-' && **str != '*' && **str != '/')
```

Comment:
```suggestion
                if (strchr("+-*/", **str) == 0)
```

### `man 3 strchr`

``` man
char *strchr(const char *s, int c);
       The strchr() and strrchr() functions return a pointer to the matched character
       or NULL if the character  is  not  found.
```

Reply:
Why does it matter? what does strchr do under the hood?

Reply:
> Why does it matter? what does strchr do under the hood?

It's a more readable to use `strchr` instead to check each single character.

Reply:
**Don't agree.**